app/client/lib/helpers/helperFunctions.js

```diff
@@ -355,3 +355,17 @@ Helpers.addInputValue = function (inputs, currentInput, formField){
             return value;
         }) || [];
 };
+
+/**
+Takes a camelcase and shows it with spaces
+
+@method toSentence
+@param {string} camelCase    A name in CamelCase or snake_case format
+@return {string} sentence    The same name with spaces
+**/
+Helpers.toSentence = function (name, noHTML) {
```

Comment:
The logic was that there are many more instances where HTML is used, as noHTML is used only inside <option> tags, and therefore the parameter can be omitted in the most common use.